### DISCUSSION TITLE
Change google-chrome pwa/webapp install name

### DISCUSSION THREAD

**Discussion by ghost** [LekoArts/gatsby-themes#618] [Questions] [answered]
Using the latest miniblog-theme I customized to my liking but in Google Chrome the install name thing (as shown in screenshot) cannot be modified (or at least I didn't find it), any tips on that ?

Thanks for your work :-) 

![Capture d’écran 2021-06-25 à 02 34 28](https://user-images.githubusercontent.com/85317794/123351948-94802400-d55e-11eb-8ef2-10493e3810c0.jpg)

**Comment by LekoArts** [comment, marked as answer]
Hi!

You'll need to change these options inside `gatsby-plugin-manifest`:

https://github.com/LekoArts/gatsby-themes/blob/7f32fd215a5b442c4ed8c046a56fa1c6334e1c6e/examples/minimal-blog/gatsby-config.js#L65-L70
